gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sun May 20 10:30:24 IST 2018
+#Fri Oct 12 22:54:02 IST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.5-all.zip
```

Comment:
Irrelevant change

Reply:
I'll fix it @iamareebjamal. And what about the UI now? Does it look good from the previous one? Or any other suggestion you'll like to give? @iamareebjamal